_includes/releases/v22.2/v22.2.0-rc.2.md

```diff
@@ -0,0 +1,21 @@
+## v22.2.0-rc.2
+
+Release Date: November 14, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-rc-2-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug which caused a migration in v22.1 to fail to drop an index on `system.statement_diagnostics_requests`. This caused upgrades from v22.1 to v22.2 to fail to create a new index with the same name as the index that was supposed to have been dropped. [#91309][#91309]
+- A nil pointer crash that could be experienced when interleaving [`SELECT FOR UPDATE SKIP LOCKED`](../v22.2/select-for-update.html) statements has been resolved. [#91257][#91257]
+- Fixed a bug present only in earlier v22.2 release candidates, in which an [`ALTER PRIMARY KEY USING COLUMNS (x, x)`](../v22.2/alter-primary-key.html) statement would result in an internal error instead of the expected user-facing error with a pgcode. [#91482][#91482]
+- Previously, certain aggregate histograms would appear in `_status/vars` but not be available for graphing in the DB Console. These are now available. They include changefeed-related histograms and row-level-TTL histograms. [#91410][#91410]
```

Comment:
suggest linking changefeeds to https://www.cockroachlabs.com/docs/v22.2/change-data-capture-overview.html

suggest linking row-level TTL to https://www.cockroachlabs.com/docs/v22.2/row-level-ttl.html